example/spec/parallel/syncGranule/SyncGranuleSuccessSpec.js

```diff
@@ -367,6 +367,55 @@ describe('The Sync Granules workflow', () => {
     });
   });
 
+  describe('when an ACL is provided in the task config and set to disabled', () => {
+    const meta = { ACL: 'disabled' };
+    let executionArn;
+    let lambdaOutput;
+    let objectACLs;
+    let files;
+    let key1;
+    let key2;
+
+    beforeAll(async () => {
+      const execution = await buildAndExecuteWorkflow(
+        config.stackName, config.bucket, workflowName, collection, provider, inputPayload, meta
+      );
+
+      executionArn = execution.executionArn;
+      console.log(`Wait for completed execution ${executionArn}`);
+
+      await waitForCompletedExecution(executionArn);
+
+      lambdaOutput = await lambdaStep.getStepOutput(executionArn, 'SyncGranule');
+      files = lambdaOutput.payload.granules[0].files;
+      key1 = s3Join(files[0].key);
+      key2 = s3Join(files[1].key);
+
+      await Promise.all([
+        s3ObjectExists({ Bucket: files[0].bucket, Key: key1 }),
+        s3ObjectExists({ Bucket: files[1].bucket, Key: key2 }),
+      ]);
+      objectACLs = await Promise.all(files.map(
+        (file) => s3().getObjectAcl({
+          Bucket: file.bucket,
+          Key: file.key,
+        })
+      ));
+    });
+
+    afterAll(async () => {
+      await deleteExecution({ prefix: config.stackName, executionArn: executionArn });
+    });
+
+    it('puts objects with ACL disabled (full control permission)', () => {
+      objectACLs.map((acl) => expect(acl.Grants[0].Permission).toBe('FULL_CONTROL'));
+    });
+
+    it('a disabled ACL configuration value in the lamba output', () => {
+      expect(lambdaOutput.meta.ACL).toBe('disabled');
+    });
+  });
+
```

Comment:
If we do allow custom ACLs should we add a test for something other than 'disabled'?

Reply:
I considered doing that as well but I wasn't sure how to actually prove (other than using the lambdaOutput) that provided ACL was used because `getObjectAcl` returns the same `FULL_CONTROL` permission for all the canned ACLs 😓  Do you think I should add one anyway? (This entire comment might be irrelevant as well since I may just want to constrict the ACL setting to be either disabled or `private`)

Reply:
What are you thoughts on this test [`d01be12` (#2946)](https://github.com/nasa/cumulus/pull/2946/commits/d01be12a10703b75204c5f22ea23328cbacb13aa#diff-6b021707aab039821866154177097c5c7c8a23911b14bf7012dc6437528e5882R424) 👀 ?

Reply:
Hmm I guess I was thinking there was a way for a user to add a non-canned ACL. Maybe that's not possible though because they'd have to write their own XML and we might not have the facility to accept it? I don't totally know what I'm talking about here.

In any case, if we do decide to allow this, these tests for other canned ACLs make sense.

Reply:
Given the change to only allow `private` or `disabled` I suspect we don't need these new tests?

Reply:
This test covers the case where a user inputs a value for ACL that isn't disabled and it supposed to check that the ACL that is set is `private` but the tricky part is, using `getObjectAcl` makes that tough to determine since it only reveals the `Permission` which is generally the same for all canned ACLs. And the lambda output is `authenticated-read` since that is what was provided to this step of the workflow. I can remove the test altogether since it's kind of confusing 😕  Let me know what you think!

Reply:
Gotchya! Thanks for that. I guess I don't really feel strongly one way or the other. It makes sense to leave these I think.